tensorboard/backend/application.py

```diff
@@ -251,7 +251,15 @@ def TensorBoardWSGIApp(
     experimental_plugins = []
     for plugin_spec in plugins:
         loader = make_plugin_loader(plugin_spec)
-        plugin = loader.load(context)
+        try:
+            plugin = loader.load(context)
+        except Exception:
+            logger.error(
+                "Failed to invoke plugin loader %s",
```

Comment:
Is this supposed to be user visible error string? Not sure how much user cares about "invoking plugin loader". I think it should say something like "Failed to load plugin %s and ignoring it".

Reply:
Yes, it’s user-facing, and you’re right, it should be improved. Took
your wording; thanks.

Reply:
(It’s slightly awkward because the actual plugin name is given by the
result of `load`, so all we have is the qualname of the loader itself,
but hopefully “WhatIfToolPluginLoader.load” is clear enough.)